src/main/java/net/F53/HorseBuff/ClientInit.java

```diff
@@ -0,0 +1,20 @@
+package net.F53.HorseBuff;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.client.keybinding.v1.KeyBindingHelper;
+import net.minecraft.client.option.KeyBinding;
+import net.minecraft.client.util.InputUtil;
+import org.lwjgl.glfw.GLFW;
+
+public class ClientInit implements ClientModInitializer {
+
+    public static KeyBinding horsePlayerInventory = KeyBindingHelper.registerKeyBinding(new KeyBinding(
+            "text.HorseBuff.keybinding.horsePlayerInventory",
+            InputUtil.Type.KEYSYM,
+            GLFW.GLFW_MOD_ALT,
```

Comment:
Try something with left alt or something, because GLFW_MOD_ALT doesn't seem to work properly.

![287867624-0fe44827-ac50-4e2c-bfa4-9ddc6afd5438](https://github.com/CodeF53/Horse-Buff/assets/37855219/1ce9f022-32c6-4e70-b106-dcf87100cecf)